src/python/py_imagebufalgo.cpp

```diff
@@ -2959,6 +2981,13 @@ declare_imagebufalgo(py::module& m)
                     "mode"_a = ImageBufAlgo::NONFINITE_BOX3,
                     "roi"_a = ROI::All(), "nthreads"_a = 0)
 
+        .def_static("normalize", &IBA_normalize, "dst"_a, "src"_a, 
+                    "incenter"_a = 0.0f, "outcenter"_a = 0.0f, "scale"_a = 1.0f,
```

Comment:
I would recommend using the exact same capitalization for parameter names for Python versus C++.